sycl/test-e2e/lit.cfg.py

```diff
@@ -525,6 +525,17 @@ def open_check_file(file_name):
 if not sycl_ls:
     lit_config.fatal("can't find `sycl-ls`")
 
+if (
+    len(config.sycl_build_targets) == 1
+    and next(iter(config.sycl_build_targets)) == "target-all"
+):
+    config.sycl_build_targets = {"target-spir"}
+    sp = subprocess.getstatusoutput(config.dpcpp_compiler + " --print-targets")
+    if "nvptx64" in sp[1]:
```

Comment:
note to myself/nobody: i was hoping we could also check for spir here, but it seems it only shows up if we build the spir-v backend which we don't do by default